src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -736,28 +771,55 @@ public static boolean backupColonyData()
             saveColonies();
         }
 
-        @NotNull final File file = getSaveLocation();
-        @NotNull final File targetFile = getBackupSaveLocation(new Date());
-        if (!file.exists())
+        try(FileOutputStream fos = new FileOutputStream(getBackupSaveLocation(new Date())))
         {
-            return true;
+            @NotNull final File saveDir = new File(DimensionManager.getWorld(0).getSaveHandler().getWorldDirectory(), FILENAME_MINECOLONIES_PATH);
+            ZipOutputStream zos = new ZipOutputStream(fos);
```

Comment:
I would be in favor of single files instead of one zip file. If that one file gets corrupt, we have the same issues all over. And touching it can be pretty risky

I suggest we have a subfolder named `colonies` and there each colony is added

That means admins can delete a colony to clear it f.e. just like mc chunks behave

Reply:
Yep much better solution

Reply:
No, you're not understanding it right, the zip file is only for the backups the colony data gets stored by each of the colonies itself they don't need to run through the colonymanager the colony manager only stores them on shut down and when he is dirty, the files are inside the minecolonies folder and they get grouped in a zip as a backup